projects/Mallard/src/helpers/push-notifications.ts

```diff
@@ -69,6 +71,7 @@ const pushNotifcationRegistration = () => {
             if (token) {
                 maybeRegister(token.token).catch(err => {
                     console.log(`Error registering for notifications: ${err}`)
+                    errorService.captureException(err)
```

Comment:
Nice to see Sentry getting used!

Reply:
😎

Reply:
You can thank @jeanlauliac for reminding me that `console.log` in a try catch is poor form, especially when we have an error service at our disposal.